browser/resources/settings/brave_reset_page/brave_reset_profile_dialog_behavior.ts

```diff
@@ -16,12 +14,13 @@ const BraveResetProfileDialogBehaviorImpl = {
    * @private
    */
   fixSendSettingsCheckbox: function() {
-    this.$.sendSettings.checked = false;
-    this.$.sendSettings.hidden = true;
-    this.$.sendSettings.disabled = true;
-    this.$.sendSettings.parentNode.hidden = true;
+    ;(this as any).shadowRoot!.getElementById('sendSettings').checked = false
+    ;(this as any).shadowRoot!.getElementById('sendSettings').hidden = true
+    ;(this as any).shadowRoot!.getElementById('sendSettings').disabled = true
+    ;(this as any).shadowRoot!.getElementById('sendSettings').
+      parentNode.hidden = true
```

Comment:
I couldn't find a better way to get around this that both satisfied the compiler and various presubmit checks. I also didn't want to convert this behavior into a mixin, though I'm not opposed to doing that down the line if it would help.

Reply:
Yeah unfortunately that's how `prettier` wants things formatted - personally I think its pretty horrendous but I guess that's why we use an opinionated tool :laughing:

Reply:
Actually, thinking about it you might be able to do something like this:


```ts
    fixSendSettingsCheckbox: function(this: HTMLElement) {
```

or if you're really lucky:

```ts
  fixSendSettingsCheckbox: function(this: { $: {
    sendSettings: HTMLInputElement /* or whatever it is */
  } }) {
```

or failing that:


```suggestion
    const sendSettings = (this as any).shadowRoot!.getElementById('sendSettings')
    sendSettings.checked = false
    sendSettings.hidden = true
    sendSettings.disabled = true
    sendSettings.parentNode.hidden = true
```

Reply:
Had to go with the last option, but it's still much nicer than what I had https://github.com/brave/brave-core/pull/27107/commits/0054c6ec6d3d7b9fc8731c6173a8357e1285f8e9